backend/connection.go

```diff
@@ -312,65 +312,65 @@ type Frame struct {
 }
 
 func (c *Connection) watchFile(action Action) {
```

Comment:
While I fully agree that a Nomad version check is necessary. Is it really necessary to comment out the code? If people are using an older version it will just throw an error that it's unable to stream the file. There currently is no implementation that takes care of older Nomad API's. If people are using 0.4.1 they have nothing now.